code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/lb/LoadBalancerConfigSetListenersActionHandler.java

```diff
@@ -46,7 +54,7 @@ public HandlerResult handle(ProcessState state, ProcessInstance process) {
             // create a new set
             createNewListenerMaps(config, newListenerIds);
         }
-        return null;
+        return objectManager.reload(config);
```

Comment:
@sonchang  @cjellick  and you are both right! and I need to be more consistent for sure :)